common/src/primitives/amount.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2022 RBB S.r.l
```

Comment:
Btw: should these years be just updated (as proposed here) or should it be a range like `2022 - 2023`?
Also all headers should be updated (not for this PR).

Reply:
Ranges vs static value - good question.

As for all headers, I didn't want to touch all files as this was a small change, but I guess I could do it for all once the "range vs static value" is sorted.

Reply:
Just checked and we already have ranges in some of the older files. E.g. `crypto/src/hash/mod.rs` has `2021-2022`, so let's stick with that. No need to bother updating all headers in this PR.